tests/test_ws_router.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, FastAPI
+from starlette.testclient import TestClient
+from starlette.websockets import WebSocket
+
+router = APIRouter()
+app = FastAPI()
+
+
+@app.websocket_route("/")
+async def index(websocket: WebSocket):
+    await websocket.accept()
+    await websocket.send_text("Hello, world!")
+    await websocket.close()
+
+
+@router.websocket_route("/router")
+async def routerindex(websocket: WebSocket):
+    await websocket.accept()
+    await websocket.send_text("Hello, router!")
+    await websocket.close()
+
+
+app.include_router(router)
```

Comment:
Can you help me creating another router that we can include with `app.include_router(router, prefix="/something")`?

And then the equivalent test...

This would help us make sure that the prefix functionality still works, and especially, that we don't break it in any future code change.

Reply:
Done.

Please note that you have to have a trailing slash for the `"/"` route of the `"/prefix"` (route is `"/prefix/"`)

It seems consistent with what I experienced so far in other parts of the routing, might be confusing at first though

Reply:
Thanks!